### PR TITLE
fix 6481 with colorbars in groupby

### LINKED ISSUE #6481: vims not the same for all images of evoked.plot_image with group_by and axes dicts and a mask
See:

```
import mne

evoked = mne.read_epochs(mne.datasets.kiloword.data_path() + "/kword_metadata-epo.fif").average()
evoked.data[0, :] *= 10
group_by = mne.channels.make_1020_channel_selections(evoked.info, midline="z12")
fig, axes = plt.subplots(nrows=3)
fig.set_size_inches((8, 10))
axes = {roi: ax for roi, ax in zip(group_by, axes)}

evoked.plot_image(
    axes=axes, group_by=group_by, show_names="all", show=False,
    mask=evoked.data < 0
);
```

If you remove the mask, all images have the same vlims.

@drammock if you have time, otherwise I will try to get to it in a month or so.

### PR DESCRIPTION
**Pull request by agramfort**
closes #6481 

why this is needed is a mystery

cc @drammock

### CODE REVIEW

**Comment by jona-sassenhagen** [comment]
I wonder if we could unit test this, e.g. by fetching the lims of a plot.

File: mne/viz/utils.py [hunk 1]
```diff
@@ -2883,6 +2883,7 @@ def _plot_masked_image(ax, data, times, mask=None, yvals=None,
             im = ax.imshow(
                 np.ma.masked_where(~mask, data), cmap=cmap, **im_args)
         else:
+            ax.imshow(data, cmap=cmap, **im_args)  # see #6481
             im = ax.imshow(data, cmap=cmap, **im_args)
```
Comment (larsoner) [inline mne/viz/utils.py]: Drawing the same thing twice is pretty weird. Did you try doing a `ax.figure.draw_idle()` here instead? That would feel cleaner than plotting the same thing twice. But if it doesn't work, I can live with this.

Comment (agramfort) [reply mne/viz/utils.py]: I tried

`ax.get_figure().canvas.draw_idle()`

but it does not fix the pb.

**Comment by larsoner** [comment]
Thanks @agramfort

**Pull request merged**